<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneWindow</class>
 <widget class="QMainWindow" name="SceneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene</string>
  </property>
  <widget class="ViewerWidget" name="viewer"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Viewpoints">
    <property name="title">
     <string>&amp;Viewpoints</string>
    </property>
    <addaction name="actionPrevious_Viewpoint"/>
    <addaction name="actionNext_Viewpoint"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_Viewpoint"/>
    <addaction name="actionRemove_Current_Viewpoint"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Contents"/>
   </widget>
   <widget class="QMenu" name="menu_Scene">
    <property name="title">
     <string>&amp;Scene</string>
    </property>
    <addaction name="actionBind_Parameters"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Viewpoints"/>
   <addaction name="menu_Scene"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Scene Tool Bar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrevious_Viewpoint"/>
   <addaction name="actionNext_Viewpoint"/>
   <addaction name="actionReturn_to_Viewpoint"/>
   <addaction name="actionCreate_Viewpoint"/>
   <addaction name="separator"/>
   <addaction name="actionBind_Parameters"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open osg file...</string>
   </property>
   <property name="toolTip">
    <string>Open osg file</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionPrevious_Viewpoint">
   <property name="icon">
    <iconset resource="../zenom.qrc">
     <normaloff>:/zenom/resources/previous.png</normaloff>:/zenom/resources/previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous Viewpoint</string>
   </property>
  </action>
  <action name="actionNext_Viewpoint">
   <property name="icon">
    <iconset resource="../zenom.qrc">
     <normaloff>:/zenom/resources/next.png</normaloff>:/zenom/resources/next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next Viewpoint</string>
   </property>
  </action>
  <action name="actionCreate_Viewpoint">
   <property name="icon">
    <iconset resource="../zenom.qrc">
     <normaloff>:/zenom/resources/viewpoint.png</normaloff>:/zenom/resources/viewpoint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create Viewpoint...</string>
   </property>
  </action>
  <action name="actionRemove_Current_Viewpoint">
   <property name="text">
    <string>&amp;Remove Current Viewpoint</string>
   </property>
  </action>
  <action name="actionBind_Parameters">
   <property name="icon">
    <iconset resource="../zenom.qrc">
     <normaloff>:/zenom/resources/link.png</normaloff>:/zenom/resources/link.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bind Parameters...</string>
   </property>
  </action>
  <action name="actionReturn_to_Viewpoint">
   <property name="icon">
    <iconset resource="../zenom.qrc">
     <normaloff>:/zenom/resources/return.png</normaloff>:/zenom/resources/return.png</iconset>
   </property>
   <property name="text">
    <string>Return to Viewpoint</string>
   </property>
  </action>
  <action name="action_Contents">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewerWidget</class>
   <extends>QWidget</extends>
   <header>scene/viewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../zenom.qrc"/>
 </resources>
 <connections/>
</ui>
